DataStore2/init.lua

```diff
@@ -182,15 +182,28 @@ end
 function DataStore:SetValidator(validator)
 	assert(
 		type(validator) == "function",
-		"function expected, got "..type(validator)
+		"function expected, got " .. type(validator)
```

Comment:
Didn't notice this last time, but `typeof` is more helpful.

```suggestion
		"function expected, got " .. typeof(validator)
```